### ISSUE TITLE
"short css" syntax doesn't work with named grid lines 

### ISSUE THREAD

**Issue by k3nsei** [ben-rogerson/twin.macro#315]
1. New custom css syntax not working with valid grid css.
```
tw`
      grid grid[[stack] 1fr / min-content [stack] 1fr]
      w-screen min-h-screen
    `
```
2. emotion
3. https://codesandbox.io/s/focused-smoke-mkr0x?file=/src/components/Layout.js

**Comment by ben-rogerson** [comment]
Thanks for letting me know about this bug - perfect use for the new short css syntax too.

I'll get this patched up.

**Comment by k3nsei** [comment]
Maybe escaping would be good idea to have `customProp[\[value\] foo bar]`

**Comment by ben-rogerson** [comment]
> Maybe escaping would be good idea to have `customProp[\[value\] foo bar]`
> 

Yeah if it's needed - if we can get away without it then that would be sweet

**Comment by ben-rogerson** [comment]
Fixed in 2.2.1 🎉 

You can also now use dash-case instead of camelCase for the css properties.
